Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -360,6 +360,15 @@ class MiscellaneousTestCase: XCTestCase {
       #endif
     }
 
+    func testCodeCoverageCompile() throws {
+        fixture(name: "Miscellaneous/CodeCoverage") { prefix in
```

Comment:
Wrap this test body in `#if os(macOS)` like the above test. Also run `.build/x86_64-apple-macosx10.10/debug/spm test --generate-linuxmain` to update the linux main entires.